### ISSUE TITLE
"All documentation" links need to become more visible on a mobile device 

### ISSUE THREAD

**Issue by Alagts** [TYPO3-Documentation/render-guides#853]
Currently when clicking the "Menu" button the menu "all documentation" is attached donwstairs after the normal menu. 
![All_documentation_location_mobile](https://github.com/user-attachments/assets/7937644a-1f30-4436-9fd8-bc159e5e452d)

It is harder to find there. We need to put it at the top, above the normal menu. How to do this needs to be discussed. My suggestion is to put it out and create an extra button that opens this menu separately from the normal menu.

**Comment by CybotTM** [comment]
The requested change — "put it at the top, above the normal menu" — has been implemented.

PR #966 (commit d2318809, merged 2025-04-21) moved the `all-documentation-menu-mobile` include from below to above the `main_menu` block in `structure/document.html.twig`. It is the only change to that template since this issue was filed.

Confirmed in the rendered HTML output (current renderer): the all-documentation menu element precedes the normal menu in the DOM:
```
108: <all-documentations-menu-mobile ...>
109:   ...
110: <div ... class="main_menu" ...>
```

So on mobile the "All documentation" menu now appears at the top, above the normal page menu.

The separate-button variant mentioned as a suggestion in the report was not implemented — the menu was repositioned rather than split into its own toggle. If the top placement is sufficient this can be closed; if a dedicated button is still wanted, that part remains open.

**Comment by CybotTM** [comment]
<img width="200"  alt="Image" src="https://github.com/user-attachments/assets/5f0349d0-2e43-45c0-a243-61f9e14e6217" />

<img width="200" alt="Image" src="https://github.com/user-attachments/assets/4e33a619-9872-4505-9675-93249c5438b8" />

**Comment by garvinhicking** [comment]
Hhhhm I must say I think "All documenation" is LESS important than the currently viewed docs. IMO first the docs links must appear.

**Comment by CybotTM** [comment]
mhm ... but this issue ""All documentation" links need to become more visible on a mobile device" is resolved ... ;-) @garvinhicking 

so we either re-scope this issue or close it and open a new one. Anyway, I cannot decide on this.

**Comment by garvinhicking** [comment]
Yeah of course - this was more feedback to the issue itself. I am just one voice and I'm happy for how you/others see this.

**Comment by linawolf** [comment]
The mobile "All documentation" menu has sat above the main menu since #966 —
`structure/document.html.twig` includes `all-documentation-menu-mobile`
before the `main_menu` block. That is what this issue asked for.

Not done: the separate toggle button suggested here, and @garvinhicking
argues above for the opposite ordering. Both are worth their own issue with
a fresh look rather than staying on this one.
